app/inputs/concerns/array_aware_input.rb

```diff
@@ -65,7 +65,7 @@ def array_of_inputs(wrapper_options = {})
   end
 
   def include_field_template(element_fields, array_options)
-    element_field_template = CGI.escapeHTML(element_fields.first.gsub(%(_0"), %(_[[index]]")))
+    element_field_template = CGI.escapeHTML(element_fields.first.gsub(%(_0"), %(_[[index]]")).gsub(/value=".*?"/,''))
```

Comment:
Let's make the template more consistent with default input rendering by substituting with a blank value attribute, i.e.
`.gsub(/ value="[^"]+"/, ' value=""'))`